upcoming-release-notes/241.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
+authors: [kyrias]
+---
+
+Stop sending `end_date` to today when downloading GoCardless transactions.
```

Comment:
```suggestion
Stop sending optional `end_date` parameter to GoCardLess which is default set to `today()` when downloading transactions from GoCardless, since it filters out credit card transactions using `valueDate` for indicating when the interest-free period for the transaction ends. [Example issue for Bank on Norwegian Sweden](https://github.com/actualbudget/actual-server/pull/237#pullrequestreview-1563979522)
```